heartcheck.gemspec

```diff
@@ -17,17 +17,22 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency 'rack', '>= 1.4.0', '< 2.1'
+  spec.add_runtime_dependency 'rack', '~> 2.2'
+
   spec.add_runtime_dependency 'multi_json', '~> 1.0'
   spec.add_runtime_dependency 'net-telnet', '~> 0.1.1'
   spec.add_runtime_dependency 'sys-uname', '~> 1.0', '>= 1.0.3'
 
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'pry-nav', '~> 0.2.4'
```

Comment:
what about adding the required ruby version?
```ruby
spec.required_ruby_version = '>= 2.3'
```


more details: https://guides.rubygems.org/specification-reference/#required_ruby_version

Reply:
didn't know that, seems really usefull...adding o/